rest/adminapitest/admin_api_test.go

```diff
@@ -2141,6 +2141,46 @@ func TestHandleCreateDB(t *testing.T) {
 	rest.RequireStatus(t, resp, http.StatusBadRequest)
 }
 
+func TestHandleCreateDBNoJsonName(t *testing.T) {
```

Comment:
These tests could be in the same test and then use sub tests to run multiple scenarios such as using the same path and name in the body, a different path and name, and a path but no name.

See `TestPutIDRevMatchBody` for an example of using sub tests.